src/Appwrite/Platform/Tasks/Backup.php

```diff
@@ -0,0 +1,252 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Exception;
+use Utopia\DSN\DSN;
+use Utopia\Platform\Action;
+use Utopia\App;
+use Utopia\CLI\Console;
+use Utopia\Pools\Group;
+use Utopia\Storage\Device;
+use Utopia\Storage\Device\DOSpaces;
+use Utopia\Storage\Device\Local;
+use Utopia\Validator\Text;
+
+class Backup extends Action
+{
+    protected ?DSN $dsn = null;
+    protected ?string $database = null;
+    public const BACKUPS = '/backups';
+    public const BACKUP_INTERVAL = 60 * 60 * 4; // 4 hours;
+    public const COMPRESS_ALGORITHM = 'lz4';
+    public const CNF = '/etc/my.cnf';
+    public const PROCESSORS = 4;
+
+    public static function getName(): string
+    {
+        return 'backup';
+    }
+
+    public function __construct()
+    {
+        $this->checkEnvVariables();
+
+        $this
+            ->desc('Backup a DB')
+            ->param('database', null, new Text(20), 'passed from command')
```

Comment:
Let's add a better description for the parameter

Reply:
Done!